src/components/InputPassword.js

```diff
@@ -126,23 +126,15 @@ class InputPassword extends Component {
               text="lower"
               open={this.state.showRequirements}
             >
-              <span
-                valid={
-                  this.state.password.toUpperCase() !== this.state.password
-                }
-              />
+              <span valid={this.state.password.toUpperCase() !== this.state.password} />
```

Comment:
This might've been me before, but if this span is not styled component, why does it need `valid prop` ?